examples/notebooks/beam-ml/bigtable_enrichment_transform.ipynb

```diff
@@ -0,0 +1,854 @@
+      ],
+      "metadata": {
+        "id": "-Lvo8O2V-0Ey"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "The following example demonstrates the code needed to add this transform to your pipeline.\n",
+        "\n",
+        "\n",
+        "```\n",
+        "with beam.Pipeline() as p:\n",
+        "  output = (p\n",
+        "            ...\n",
+        "            | \"Enrich with BigTable\" >> Enrichment(bigtable_handler, timeout=10)\n",
+        "            | \"RunInference\" >> RunInference(model_handler)\n",
+        "            ...\n",
+        "            )\n",
+        "```\n",
+        "\n",
+        "\n",
+        "\n",
+        "\n"
+      ],
+      "metadata": {
+        "id": "xJTCfSmiV1kv"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "To make a prediction, use the following fields: `product_id`, `quantity`, `price`, `customer_id`, and `customer_location`. Retrieve the `customer_location` from the Bigtable.\n",
+        "\n",
+        "Because the enrichment transform performs a [`cross_join`](https://beam.apache.org/releases/pydoc/current/apache_beam.transforms.enrichment.html#apache_beam.transforms.enrichment.cross_join) by default, design the custom join to enrich the input data. This design ensures that the join includes only the specified fields."
+      ],
+      "metadata": {
+        "id": "F-xjiP_pHWZr"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "def custom_join(left: Dict[str, Any], right: Dict[str, Any]):\n",
+        "  enriched = {}\n",
+        "  enriched['product_id'] = left['product_id']\n",
+        "  enriched['quantity'] = left['quantity']\n",
+        "  enriched['price'] = left['price']\n",
+        "  enriched['customer_id'] = left['customer_id']\n",
+        "  enriched['customer_location'] = right['demograph']['customer_location']\n",
+        "  return beam.Row(**enriched)"
+      ],
+      "metadata": {
+        "id": "8LnCnEPNIPtg"
+      },
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "## Use `PyTorchModelHandlerTensor` to run inference\n",
+        "\n"
+      ],
+      "metadata": {
+        "id": "CX9Cqybu6scV"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "Because the enrichment transform outputs data in the format `beam.Row`, to make it compatible with [`PyTorchModelHandlerTensor`](https://beam.apache.org/releases/pydoc/current/apache_beam.ml.inference.pytorch_inference.html#apache_beam.ml.inference.pytorch_inference.PytorchModelHandlerTensor),  convert it to `torch.tensor`. Additionally, the enriched field `customer_location` is a `string`, but the model requires a `float`. Convert `customer_location` to a `float`."
```

Comment:
```suggestion
        "Because the enrichment transform outputs data in the format `beam.Row`, to make it compatible with the [`PyTorchModelHandlerTensor`](https://beam.apache.org/releases/pydoc/current/apache_beam.ml.inference.pytorch_inference.html#apache_beam.ml.inference.pytorch_inference.PytorchModelHandlerTensor) interface,  convert it to `torch.tensor`. Additionally, the enriched field `customer_location` is a `string` type, but the model requires a `float` type. Convert the `customer_location` field to a `float` type."
```